filters/accesslog/log_test.go

```diff
@@ -0,0 +1,25 @@
+package accesslog
+
+import (
+	"testing"
+
+	"github.com/zalando/skipper/filters/filtertest"
+)
+
+func Test(t *testing.T) {
+	const state = "false"
+
+	f, err := NewAccessLog().CreateFilter([]interface{}{state})
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	var ctx filtertest.Context
+	ctx.FStateBag = make(map[string]interface{})
+
+	f.Request(&ctx)
+	bag := ctx.StateBag()
+	if bag[AccessLogEnabledKey] != false {
```

Comment:
can you also test the negative?
This test would also be fine if this is the default.

Reply:
sure, I'll update the test and also add the docs as you mentioned.